src/ray/object_manager/plasma/store.cc

```diff
@@ -161,25 +167,28 @@ PlasmaError PlasmaStore::HandleCreateObjectRequest(const std::shared_ptr<Client>
                    << ", metadata_size=" << object_info.metadata_size;
   }
   //TODO(Jae) Erase this later
-  /*
   const int64_t footprint_limit = allocator_.GetFootprintLimit();
-  const float allocated_percentage =
-        static_cast<float>(allocator_.Allocated()) / footprint_limit;
-  if(allocated_percentage >= block_tasks_threshold_){
-	  blockTasks();
+  float allocated_percentage;
```

Comment:
I thought we were going to do the thresholding in a separate PR? In this PR, I think we can always assume that we should block/evict tasks if object creation is blocked by OOM.